spec/calendar.html

```diff
@@ -38,14 +38,27 @@ <h1>GetISO8601Calendar ( )</h1>
     <emu-clause id="sec-temporal-calendarfields" aoid="CalendarFields">
       <h1>CalendarFields ( _calendar_, _fieldNames_ )</h1>
       <p>
-        The CalendarFields abstract operation transforms a List of String values _fieldNames_ into another List of String values by calling the `fields` method of the given _calendar_ Object, falling back to the intrinsic operation if not present.
+        The CalendarFields abstract operation transforms a List of String values _fieldNames_ into another List of String values by calling the `fields` method of the given _calendar_ Object, falling back to a default operation if not present.
       </p>
       <emu-alg>
         1. Let _fields_ be ? Get(_calendar_, *"fields"*).
```

Comment:
should this use GetMethod, to ensure it's either nullish or a function?

Reply:
Agreed, this is another abstract op that I wasn't familiar with. I think it should be used consistently throughout the spec text, so I've opened #1323 for that.